scripts/export_models_ops_correlation.py

```diff
@@ -117,9 +213,9 @@ def create_sheet_from_exported_models_unique_op_configuration(
                 dumped_op_configuration_file_path = os.path.join(model_path, dumped_op_configuration_file)
                 op_config_df = pd.read_csv(
                     dumped_op_configuration_file_path,
-                    sep="-",
+                    sep=os.getenv("FORGE_EXPORT_UNIQUE_OP_CONFIG_CSV_DELIMITER", "/"),
```

Comment:
Why we are defining delimiter through env variable? Is it expected to change, and if yes, why?

Reply:
I can give more context here.

Using a configurable delimiter provides flexibility in this logic for different use cases. For example:
- Using ',' will outline issues for how shapes are interpreted
- Using '-' will outline issues around the usage of negative dims

Having unknowns around op specifics, it's more flexible to use a configurable delimiter. When this logic matures, we can remove configurable part, and just use delimiter we see fit for all cases :))